### PR TITLE
[WIP] CRUD enhancement

### PR DESCRIPTION
**Pull request by ucabaur**

### CODE REVIEW

File: generators/crud/templates/crud-helpers/table-action-cell.test.js [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+
+  describe('[Event]', () => {
+    it('should navigate to the edit route of a given model when the edit button is clicked', () => {
+      const routeSpy = sinon.spy();
+      const { editButton } = setup({
+        navigateTo: routeSpy,
+        row: {
+          value: 4,
+        },
+        modelBasePath: 'ae-data',
```
Comment (cRicateau) [inline generators/crud/templates/crud-helpers/table-action-cell.test.js]: ae-data

File: generators/crud/templates/crud-helpers/table-action-cell.test.js [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+      });
+
+      editButton.simulate('click');
+      expect(routeSpy.calledWith('ae-data/edit/4')).to.be.true; // eslint-disable-line
```
Comment (cRicateau) [inline generators/crud/templates/crud-helpers/table-action-cell.test.js]: ae-data

File: generators/crud/templates/crud-helpers/table-action-cell.test.js [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+    });
+
+    it('should call the deleteElement prop when the delete button is clicked', () => {
+      const deleteSpy = sinon.spy();
+      const { deleteButton } = setup({
+        deleteElement: deleteSpy,
+        row: {
+          value: 4,
+        },
+        modelBasePath: 'ae-data',
```
Comment (cRicateau) [inline generators/crud/templates/crud-helpers/table-action-cell.test.js]: ae-data

File: generators/crud/templates/crud-helpers/table-action-cell.test.js [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+import TableActionCell from './table-action-cell';
+
+describe('[Component] TableActionCell', () => {
+  const defaultProps = {
+    row: {
+      value: 2,
+    },
+    navigateTo: () => {},
+    deleteElement: () => {},
+    modelBasePath: 'ae-data',
```
Comment (cRicateau) [inline generators/crud/templates/crud-helpers/table-action-cell.test.js]: ae-data

**Comment by cRicateau** [comment]
it should be closed

**Comment by ghamaide** [comment]
Closed - no more CRUD generator

**Pull request closed without merging**